test/uk/gov/hmrc/brm/config/FeatureSwitchSpec.scala

```diff
@@ -265,55 +264,55 @@ class FeatureSwitchSpec extends UnitSpec with BeforeAndAfter with OneAppPerTest
 
         "return false when parent feature is disabled" taggedAs Tag("disabled") in {
           implicit val payload: Payload = buildPayload(Some("123456789"), BirthRegisterCountry.ENGLAND)
-          FeatureFactory().validate() shouldBe false
+          DownstreamFeatureFactory().enabled() shouldBe false
         }
 
         "return false when parent feature is enabled but reference feature is disabled" taggedAs Tag("referencefeaturedisabled") in {
           implicit val payload: Payload = buildPayload(Some("123456789"), BirthRegisterCountry.ENGLAND)
-          FeatureFactory().validate() shouldBe false
+          DownstreamFeatureFactory().enabled() shouldBe false
         }
 
         "return true when parent feature is enabled and reference feature is enabled" taggedAs Tag("referencefeatureenabled") in {
           implicit val payload: Payload = buildPayload(Some("123456789"), BirthRegisterCountry.ENGLAND)
-          FeatureFactory().validate() shouldBe true
+          DownstreamFeatureFactory().enabled() shouldBe true
         }
 
         "return false when date of birth is invalid" taggedAs Tag("enabled") in {
           implicit val payload: Payload = buildPayload(Some("123456789"), BirthRegisterCountry.SCOTLAND, "2005-02-03")
-          FeatureFactory().validate() shouldBe false
+          DownstreamFeatureFactory().enabled() shouldBe false
         }
 
         "return true when date of birth is invalid but switch is off" taggedAs Tag("referencefeatureenabled") in {
           implicit val payload: Payload = buildPayload(Some("123456789"), BirthRegisterCountry.SCOTLAND, "2005-02-03")
-          FeatureFactory().validate() shouldBe true
+          DownstreamFeatureFactory().enabled() shouldBe true
         }
 
       }
 
       "reference number NOT provided" should {
         "return false when parent feature is disabled" taggedAs Tag("disabled") in {
           implicit val payload: Payload = buildPayload(None, BirthRegisterCountry.SCOTLAND)
-          FeatureFactory().validate() shouldBe false
+          DownstreamFeatureFactory().enabled() shouldBe false
         }
 
         "return false when parent feature is enabled but detail feature is disabled" taggedAs Tag("detailfeaturedisabled") in {
           implicit val payload: Payload = buildPayload(None, BirthRegisterCountry.SCOTLAND)
-          FeatureFactory().validate() shouldBe false
+          DownstreamFeatureFactory().enabled() shouldBe false
         }
 
         "return true when parent feature is enabled and detail feature is enabled" taggedAs Tag("detailfeatureenabled") in {
           implicit val payload: Payload = buildPayload(None, BirthRegisterCountry.SCOTLAND)
-          FeatureFactory().validate() shouldBe true
+          DownstreamFeatureFactory().enabled() shouldBe true
```

Comment:
@adamconder Yeh, this one isn't used it seems (you've moved/updated it in config/ now, right?)